tests/test_irr_parse.py

```diff
@@ -17,10 +17,42 @@ def test_parse_basic(tmp_path):
     result_file = Path(context.out_dir_irr) / "irr_final.txt"
     assert result_file.exists()
 
-    content = []
     with open(str(result_file), 'r') as f:
-        for l in f.readlines():
-            content.append(l.strip())
-    assert content == ["193.254.30.0/24 AS12726", "212.166.64.0/19 AS12321", "212.80.191.0/24 AS12541"]
-    # removed duplicate network with higher ASN
-    assert "212.80.191.0/24 AS12542" not in content
+        content = [ l.strip() for l in f.readlines() ]
+    assert content == ["193.254.30.0/24 AS12726", "212.166.64.0/19 AS12321", "212.80.191.0/24 AS12541", "212.16.0.0/24 AS12346", "212.17.0.0/24 AS12347", "2345:2ca::/32 AS12345" ]
```

Comment:
Hm, the test below checks all of these individually, right? You could just move the final check into the test below at the end and say "check that no unintended additional entries exist" because I guess that's what this check is still good for. But I don't think we need to do the same parsing twice just for that.

Reply:
yea good point, fixed.

Reply:
But why keep this test around at all then? The other test implicitly tests that the file exists, too, right? I think you can remove the test and just move the `exists()` check into the test below. This saves us some runtime and lines of code by not doing the parsing twice :)

Reply:
fair enough. just figured they thematically test different things -- the file vs its content, but i get the runtime concern.